### PR TITLE
topic2KeyPoint5

### PR DESCRIPTION
**Pull request by leandrodalbo**
Hello again, I am making the pull request for the key point 5 of the topic 2. I'll be waiting for your corrections. 

Looking forward to your response.

### CODE REVIEW

File: topic2KeyPoint5/topic2KeyPoint1AndKeyPoint2/bin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (tomiok) [inline topic2KeyPoint5/topic2KeyPoint1AndKeyPoint2/bin/pom.xml]: Just one pom for project... prob u commit this by error.

File: topic2KeyPoint5/pom.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
```
Comment (tomiok) [inline topic2KeyPoint5/pom.xml]: try to use JUnit version 4.x.x in order to use Annotations in test cases. ! 

This is OK, but is not used any more.

Comment (leandrodalbo) [reply topic2KeyPoint5/pom.xml]: Done.

File: topic2KeyPoint5/topic2KeyPoint1AndKeyPoint2/bin/pom.xml (hunk 1 above)
Comment (leandrodalbo) [reply topic2KeyPoint5/topic2KeyPoint1AndKeyPoint2/bin/pom.xml]: Done. I don't know how it could finished right there.

**Pull request merged**